gno.land/cmd/gnoweb/static/css/app.css

```diff
@@ -236,15 +236,21 @@ a[href="#"] {
 
 ul {
   padding: 0;
+  list-style: none;
+}
+
+ul > li > ul {
+    padding-left: 1rem;
+    list-style: circle inside;
```

Comment:
cc @alexiscolin: I think this is a good bugfix, but since you're working on the same file on your PR #781; please make a quick review.

Reply:
Yes, nice UX improvement for nested unordered lists! 👍